Kernel/Arch/x86/SIMDState.h

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (c) 2022, Leon Albrecht <leon.a@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/EnumBits.h>
+#include <AK/FPControl.h>
+#include <AK/Platform.h>
+#include <AK/Types.h>
+
+namespace Kernel::SIMD {
+
+// Intel-Manual Vol 1 Chp 13.4
+enum StateComponent : u64 {
+    X87 = 1ull << 0,
+    SSE = 1ull << 1u, // xmm0-xmm7(15)
```

Comment:
the mixing of literal suffixes here is a bit confusing. why is 1 the only one with a suffix? and why is the suffix not ull to match the 1 on the left side? if we keep em i'd rather use the all caps version too, but that's style

Reply:
I'll add suffixes to the rest of these as well. I didn't notice this when I copy pasted this.